frontends/ast/ast.cc

```diff
@@ -1152,6 +1152,8 @@ static AstModule* process_module(AstNode *ast, bool defer, AstNode *original_ast
 void AST::process(RTLIL::Design *design, AstNode *ast, bool dump_ast1, bool dump_ast2, bool no_dump_ptr, bool dump_vlog1, bool dump_vlog2, bool dump_rtlil,
 		bool nolatches, bool nomeminit, bool nomem2reg, bool mem2reg, bool noblackbox, bool lib, bool nowb, bool noopt, bool icells, bool pwires, bool nooverwrite, bool overwrite, bool defer, bool autowire)
 {
+    current_scope.clear();
```

Comment:
Use tabs to indent rather than four spaces.

Reply:
Oh, you're completely right, my bad!